GLTFSDK/Source/MeshPrimitiveUtils.cpp

```diff
@@ -930,20 +732,9 @@ std::vector<uint32_t> MeshPrimitiveUtils::GetJointWeights32(const Document& doc,
         throw GLTFException("Invalid type for weights accessor " + weightsAccessor.id);
     }
 
-    switch (weightsAccessor.componentType)
```

Comment:
As with the color data, joint weight attributes using unsigned int types should be converted into float and back into ints